k8s/k8s.go

```diff
@@ -4487,6 +4491,29 @@ func (k *k8sOps) UpdateBackupLocation(backupLocation *v1alpha1.BackupLocation) (
 	return k.storkClient.Stork().BackupLocations(backupLocation.Namespace).Update(backupLocation)
 }
 
+func (k *k8sOps) ValidateBackupLocation(name, namespace string, timeout, retryInterval time.Duration) error {
+	t := func() (interface{}, bool, error) {
+		if err := k.initK8sClient(); err != nil {
+			return "", true, err
+		}
+
+		resp, err := k.GetBackupLocation(name, namespace)
+		if err != nil {
+			return "", true, err
+		}
+		return "", true, &ErrFailedToValidateCustomSpec{
```

Comment:
Need to return `"", false, nil` if err is nil. This will always return error right now.

Reply:
Fixed.